plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/LocalCheInfrastructureProvisioner.java

```diff
@@ -35,19 +38,23 @@
     private final WorkspaceFolderPathProvider workspaceFolderPathProvider;
     private final WindowsPathEscaper          pathEscaper;
     private final String                      projectFolderPath;
-	private final String                      volumesOptions;
+	private final String                      projectsVolumeOptions;
```

Comment:
Please replace tab character with spaces. Someone put it here but it is not allowed for indentation in Che.